### DISCUSSION TITLE
Welcome to sharp Discussions!

### DISCUSSION THREAD

**Discussion by dvlpp** [code16/sharp#365] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DLUK44** [comment]
Trying to follow the tutorial here https://sharp.code16.fr/docs/guide/building-menu.html#write-and-declare-the-class and it says to add ````"menu" => MySharpMenu::class```` to the ````sharp.php```` config file. When I do, I get the error ````Target class [app\Sharp\MySharpMenu] does not exist````, it does exist, it's in ````app\Sharp````. Is that the correct location? There doesn't seem to be a generator for the SharpMenu class and the documentation doesn't seem to say where to create the file so I just guessed. I have tried moving it around in case the location is hardcoded somewhere, I have also tried changing the line to ````"menu" => app\Sharp\MySharpMenu::class,```` etc. but still the same error.

I've followed the documentation twice now and end up at this same point. Any ideas?

**Reply by dvlpp** [reply]
Well, you can create you Menu class anywhere you like, as long as you report the full path on the config file. Based on your error, I suspect that you don't have the correct namespace in your SharpMenu file.
